Components/Badge/badge.html

```diff
@@ -5,32 +5,101 @@
+						</li>
+						<li class="main-nav-items"><a href="../Alert/alert.html">Avatar</a></li>
+						<li class="main-nav-items current-pg"><a href="#">Badge</a></li>
+						<li class="main-nav-items"><a href="../Button/button.html">Button</a></li>
+						<li class="main-nav-items"><a href="../Card/card.html">Card</a></li>
+						<li class="main-nav-items"><a href="../Image/image.html">Image</a></li>
+						<li class="main-nav-items"><a href="../Input/input.html">Input</a></li>
+						<li class="main-nav-items"><a href="../Navigation/navigation.html">Navigation</a></li>
+						<li class="main-nav-items"><a href="../Rating/rating.html">Rating</a></li>
+					</ul>
+				</nav>
+			</aside>
+			<section class="main-content avatar-pg">
+				<h2>Badge</h2>
+				<p class="pg-description pg-p">
+					A badge component is used to add additional information to the content.
+				</p>
+				<h3>Avatar Badge</h3>
+				<p class="pg-p">
+					Avatar badge are used to represent if a user's status which can be either online / offline
+					/ dnd. To use add a container with <code class="inline-code">badge-container</code> class.
+				</p>
+				<div class="badge-example">
+					<div class="badge-container">
+						<img src="../../Assets/avatar-2.jpg" alt="user avatar" class="avatar avatar-lg" />
+						<span class="badge badge-online"></span>
+					</div>
+					<div class="badge-container">
+						<img src="../../Assets/avatar-2.jpg" alt="user avatar" class="avatar avatar-md" />
+						<span class="badge badge-offline"></span>
+					</div>
+					<div class="badge-container">
+						<img src="../../Assets/avatar-2.jpg" alt="user avatar" class="avatar avatar-sm" />
+						<span class="badge badge-dnd"></span>
+					</div>
+				</div>
+				<div class="code-snippet">
+					<script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fkart-c%2FCL-code-snippets%2Fblob%2Fmain%2Fbadge.html&style=vs2015&showBorder=on&showCopy=on&fetchFromJsDelivr=on"></script>
+				</div>
+				<h3>Icon Badge</h3>
+				<p class="pg-p">Icon badge are number badges on icon to show user useful information.</p>
+				<div class="badge-example">
+					<div class="badge-container icon-lg">
+						<i class="fas fa-shopping-bag"></i>
+						<span class="icon-badge">6</span>
+					</div>
+					<div class="badge-container icon-lg">
+						<i class="far fa-bell"></i>
+						<span class="icon-badge">12</span>
+					</div>
+				</div>
+
+				<div class="code-snippet last-snippet">
+					<script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fkart-c%2FCL-code-snippets%2Fblob%2Fmain%2Fbadge2.html&style=vs2015&showBorder=on&showCopy=on&fetchFromJsDelivr=on"></script>
+				</div>
+
+				<div class="main-footer">
+					<a href="../Avatar/avatar.html" class="previous-pg"
+						><i class="fas fa-arrow-left"></i> Avatar</a
+					>
+					<a href="../Button/button.html" class="next-pg"
+						>Button <i class="fas fa-arrow-right"></i
+					></a>
```

Comment:
The code indentation looks broken here. Maybe you could fix this.

Reply:
I think I missed the button page I will fix it right away